docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+version: "3.9"
+
+services:
+  task_app:
+    container_name: task_app
+    image: task_app
+    build: .
+    ports:
+      - "5000:5000"
+    environment:
+      - DB_URL=postgresql://postgres:postgres@postgres_db:5432/postgres
```

Comment:
@YanaRudyi it is always better to divide such things to smaller units like password, username, host, etc to be rewritten sepatately depending on the environment